unity-renderer/Assets/Rendering/Utils/GPUSkinning/GPUSkinningThrottlerService.cs

```diff
@@ -0,0 +1,101 @@
+using Cysharp.Threading.Tasks;
+using GPUSkinning;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GPUSkinningThrottlerService : IGPUSkinningThrottlerService
+{
+    private readonly Dictionary<IGPUSkinning, int> gpuSkinnings = new Dictionary<IGPUSkinning, int>();
+
+    private bool stopAsked;
+
+
+    public void Initialize()
+    {
+        if (stopAsked)
+        {
+            Debug.LogError("GPUSkinningThrottlerService: Initialize called while stop was asked");
+            return;
+        }
+
+#if UNITY_WEBGL && !UNITY_EDITOR
+        CoroutineStarter.Start(ThrottleUpdate());
+#else
+        ThrottleUpdateAsync();
+#endif
```

Comment:
Why are we using coroutines? WebGL can handle unitasks as well.

Reply:
So far we have not found a case where a coroutine solves something a task cannot, as a rule of thumb always go first for the task approach first.

Reply:
Yeah I agree, lets do a Task only routine